gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -480,6 +480,8 @@ def run(argv=None):
     num_partitions = 1
 
   if known_args.output_table:
+    options = pipeline_options.PipelineOptions(pipeline_args)
```

Comment:
This variable is used only once in the next line. Is it possible to combine these two lines?